CONTRIBUTORS.md

```diff
@@ -1 +1,2 @@
 ## Contributors
+cynik
```

Comment:
This doesn't seem to be your correct GitHub username. Isn't it `criticic`?

Reply:
I use both, depending on availability, I'll fix it.